src/bot/cogs/invite.py

```
@@ -0,0 +1,246 @@
+            .replace("{サーバー名}", member.guild.name)
+
+    @commands.Cog.listener()
+    async def on_member_join(self, member: discord.Member):
+        guild = member.guild
+        is_enabled = await self.bot.api.is_module_enabled(str(guild.id), "invite")
+        if not is_enabled or not is_enabled.get('enabled'):
+            return
+
+        try:
+            new_invites = await guild.invites()
+        except:
```

Ruff (0.15.7)

[error] Do not use bare `except`

(E722)